ml-agents/mlagents/trainers/trainer/rl_trainer.py

```diff
@@ -122,6 +124,23 @@ def create_tf_policy(
         """
         pass
 
+    def create_saver(self, policy: Policy) -> BaseSaver:
```

Comment:
I think this should be a static factory rather than being part of the rl_trainer.py script.

Reply:
do you think it should be moved to other files or just make it a static method?

Reply:
I would prefer to another file, but static is good enough. I try to make everything that can be static into a static method because it makes testing much easier.